py2many/python_transformer.py

```diff
@@ -0,0 +1,28 @@
+try:
+    from ast import unparse
+except ImportError:
+    from astor import to_source as unparse
+
+    # Note ast-compat and astunparse packages fail internally with:
+    # AttributeError: 'Constant' object has no attribute 'kind'
+
+    # https://github.com/isidentical/backports.ast_unparse/commit/e7b1aea is broken
+
+from .clike import CLikeTranspiler
+
+
+class PythonTranspiler(CLikeTranspiler):
+    NAME = "python"
+
+    def visit(self, tree):
+        return unparse(tree) + "\nif __name__ == '__main__': main()"
+
+    def usings(self):
+        return "\n".join(
+            [
+                "from typing import Callable, Dict, List, Set, Optional",
```

Comment:
It would be useful to post-process with `autoflake` or https://github.com/hakancelik96/unimport to remove unused imports.